website/docs/r/wafv2_rule_group.html.markdown

```diff
@@ -524,6 +525,22 @@ The `ip_set_forwarded_ip_config` block supports the following arguments:
 * `header_name` - (Required) - The name of the HTTP header to use for the IP address.
 * `position` - (Required) - The position in the header to search for the IP address. Valid values include: `FIRST`, `LAST`, or `ANY`. If `ANY` is specified and the header contains more than 10 IP addresses, AWS WAFv2 inspects the last 10.
 
+### JSON Body
+
+The `json_body` block supports the following arguments:
+
+* `invalid_fallback_behavior` - (Optional) What to do when JSON parsing fails. Defaults to evaluating up to the first parsing failure. Valid values are `EVALUATE_AS_STRING`, `MATCH` and `NO_MATCH`.
+* `match_pattern` - (Required) The patterns to look for in the JSON body. See [JSON Match Pattern](#json-match-pattern) for details.
+* `match_scope` - (Required) The parts of the JSON to match against using the `match_pattern`. Valid values are `ALL`, `KEY` and `VALUE`.
+* `oversize_handling` - (Optional) What to do if the body is larger than can be inspected. Valid values are `CONTINUE` (default), `MATCH` and `NO_MATCH`.
+
+### JSON Match Pattern
+
+The `match_pattern` block supports the following arguments. You must specify either `all` or `included_paths`, but not both.
+
+* `all` - (Optional) Match all the elements.
+* `included_paths` - (Optional) Match only the specified paths. Paths are defined using [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901) syntax (e.g., `/dogs/0/name`).
+
```

Comment:
I think this could be removed. I made a suggestion above on how we can point people to the correct AWS API reference